### PR TITLE
Scripts to check issue 1917 

### PR DESCRIPTION
**Pull request by mkorniichuk**
ATF Test Scripts to check [#1917](https://github.com/SmartDeviceLink/sdl_core/issues/1917)
This PR is **not ready** for review.
This is **draft** version of the script.

### Summary
A test script to verify that issue is fixed

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by dboltovskyi** [comment]
Livio team please do not review this PR for now since it hasn't passed our internal review yet.  
Review and appropriate updates going be done till 1st of November.
Thank you.

**Comment by GetmanetsIrina** [comment]
PR is rebased on current develop(8b2019d) branch.
Script is applicable for all policy flows and transport types.
Script has been moved to 6_1 folder.
Test set for script is added.

File: test_scripts/Defects/6_1/1917_Media_navigation_and_nonmedia_apps_must_be_activated_during_active_embedded_audio_source.lua [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+end
+
+local function deactivateApp(pAppId)
+  if not pAppId then pAppId = 1 end
+  common.getHMIConnection():SendNotification("BasicCommunication.OnAppDeactivated",
+    { appID = common.getHMIAppId(pAppId) })
+  common.getMobileSession(pAppId):ExpectNotification("OnHMIStatus")
+  :ValidIf(function(_, data)
+    if config["application"..pAppId].registerAppInterfaceParams.isMediaApplication == true or
+      config["application"..pAppId].registerAppInterfaceParams.appHMIType[1] == "NAVIGATION"
```
Comment (jacobkeeler) [inline test_scripts/Defects/6_1/1917_Media_navigation_and_nonmedia_apps_must_be_activated_during_active_embedded_audio_source.lua:84]:
```suggestion
      config["application"..pAppId].registerAppInterfaceParams.appHMIType[1] == "NAVIGATION"
```
`NAVIGATION` app is `STREAMABLE` from `LIMITED`

File: test_scripts/Defects/6_1/1917_Media_navigation_and_nonmedia_apps_must_be_activated_during_active_embedded_audio_source.lua [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+  end)
+end
+
+local function embededAudioActivated()
+  common.getHMIConnection():SendNotification("BasicCommunication.OnEventChanged",
+    { eventName = "AUDIO_SOURCE", isActive = true})
+  common.getMobileSession(1):ExpectNotification("OnHMIStatus",
+    { hmiLevel = "BACKGROUND", audioStreamingState = "NOT_AUDIBLE", videoStreamingState = "NOT_STREAMABLE" })
+  common.getMobileSession(2):ExpectNotification("OnHMIStatus",
+    { hmiLevel = "BACKGROUND", audioStreamingState = "NOT_AUDIBLE", videoStreamingState = "NOT_STREAMABLE" })
```
Comment (jacobkeeler) [inline test_scripts/Defects/6_1/1917_Media_navigation_and_nonmedia_apps_must_be_activated_during_active_embedded_audio_source.lua:105]:
```suggestion
  -- hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE", videoStreamingState = "STREAMABLE"
  common.getMobileSession(2):ExpectNotification("OnHMIStatus"):Times(0)
```
App would be in `LIMITED`, `AUDIBLE`, `STREAMABLE`

File: test_scripts/Defects/6_1/1917_Media_navigation_and_nonmedia_apps_must_be_activated_during_active_embedded_audio_source.lua [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    { hmiLevel = "FULL", audioStreamingState = "AUDIBLE" })
+  :Times(2)
+  :Do(function(exp)
+    if exp.occurences == 1 then
+      local requestId = common.getHMIConnection():SendRequest("SDL.ActivateApp", { appID = common.getHMIAppId(1) })
+      EXPECT_HMIRESPONSE(requestId)
+    end
+  end)
+  common.getMobileSession(2):ExpectNotification("OnHMIStatus",
+    { hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE" })
```
Comment (jacobkeeler) [inline test_scripts/Defects/6_1/1917_Media_navigation_and_nonmedia_apps_must_be_activated_during_active_embedded_audio_source.lua:129]:
```suggestion
  -- hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE", videoStreamingState = "STREAMABLE"
  common.getMobileSession(2):ExpectNotification("OnHMIStatus"):Times(0)
```
App would be in `LIMITED`, `AUDIBLE`, `STREAMABLE`

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

**Comment by jacobkeeler** [comment]
Closing in favor of #2452

**Pull request closed without merging**